### PR TITLE
fix(SCT-661): display warning notes for all users

### PR DESCRIPTION
**Pull request by chrishutchinson**
**What**  
We did a fix to ensure practitioners could see warning notes for their residents last week. This extends that change to show warning notes for all users (but only users with the right permissions can `review / end` them).

**Why**  
[SCT-661](https://hackney.atlassian.net/browse/SCT-661)

### CODE REVIEW

File: components/WarningNote/WarningNotes.spec.tsx [hunk 1]
```diff
@@ -13,33 +28,44 @@ jest.mock('next/router', () => ({
...
-    expect(asFragment()).toMatchSnapshot();
-  });
-
-  it('should not render the review date if there is a scheduled future review date', () => {
+  beforeEach(() => {
     jest.spyOn(warningNotes, 'useWarningNotes').mockImplementation(() => ({
       data: [
         warningNoteFactory.build({
-          nextReviewDate: '2020-11-12',
+          nextReviewDate: '2020-12-30',
```
Comment (martapederiva) [inline components/WarningNote/WarningNotes.spec.tsx:35]: Why the date change?

Comment (Renny4Real) [inline components/WarningNote/WarningNotes.spec.tsx:35]: Why do we change the date here? e.g. preference? Date weirdness in tests? 😁

**Review by Renny4Real — APPROVED** [review]
Looks good 🎉

File: components/WarningNote/WarningNotes.spec.tsx (hunk 1 above)
Comment (chrishutchinson) [reply components/WarningNote/WarningNotes.spec.tsx:35]: @Renny4Real: @martapederiva had exactly the same question, and I didn't have a good answer 😅 . I'm going to take a look and will report back, it's possible I changed it in error!

Comment (chrishutchinson) [reply components/WarningNote/WarningNotes.spec.tsx]: @Renny4Real @martapederiva: Looked like an oversight. Restoring it to the original date, things are still working as expected 👍

**Pull request merged**